src/licensedcode/data/licenses/bsd-3-clause-hp.LICENSE

```diff
@@ -0,0 +1,73 @@
+---
+key: bsd-3-clause-hp
+short_name: Hewlett-Packard BSD variant license
+name: Hewlett-Packard BSD variant license
+category: Permissive
+owner: HP - Hewlett Packard
+notes: |
```

Comment:
bsd-3-clause-hp.LICENSE:

This just has HP over a bsd-new?

Reply:
@AyanSinhaMahapatra  your point is valid, I think, although there might be other minor differences, but please go ahead and create this one, since we really need to stay in synch with the SPDX list.  Be sure to include the HP references in the text.